### DISCUSSION TITLE
element的select如何实现在弹出选项框最下面加一个自定义的内容

### DISCUSSION THREAD

**Discussion by ChenKun1997** [ElemeFE/element#22476] [Q&A]
![image](https://user-images.githubusercontent.com/24249578/230346088-e477dc63-9857-4abc-914c-e0fd11992b76.png)
如上图，如何在最下面加一个”管理环境“，他不属于option，我直接在<el-option>元素下面写个div确实可以，如图
![image](https://user-images.githubusercontent.com/24249578/230347148-bfe8b0dc-5819-4e3a-a319-204a351372e7.png)

但是当这个select可搜索时，
![image](https://user-images.githubusercontent.com/24249578/230347272-d5a04c4a-cd43-421e-b06b-aad6189a544b.png)
搜不到数据时这个无匹配数据竟然在下面，有没有最佳实践啊

**Comment by wangdaodao** [comment]
我们改了源码，加了一个插槽解决的…
